### ISSUE TITLE
Bug: CPU plugin label can't be unset/blanked

### ISSUE THREAD

**Issue by pataquets** [danerwilliams/tmux-dracula#200]
### Describe the bug
Setting `@dracula-cpu-usage-label` to `""` fails to unset CPU plugin's label.
Also unsetting the variable with `set -gu @dracula-cpu-usage-label` does not work.

### To Reproduce
1. `set -g @dracula-cpu-usage-label ""`
2. Reload `tmux`
3. Label is set to default `CPU`.

### Expected behavior
Label should be blankable to save statusbar estate. I guess that the current default when blank is not preserving previous behavior.

### System
 - OS: Ubuntu Bionic
- Tmux Version: 2.6

**Comment by ethancedwards8** [comment]
I think this is due to the internal implementation of get_tmux_option():

```
get_tmux_option() {
  local option="$1"
  local default_value="$2"
  local option_value="$(tmux show-option -gqv "$option")"
  if [ -z "$option_value" ]; then
    echo "$default_value"
  else
    echo "$option_value"
  fi
}
```

its checking to see if the label is empty with `-z`. When you set it to `""`, it is "empty" for bashes purposes. I'll think about a better way to do this.

**Comment by ethancedwards8** [comment]
Yeah that doesn't work lol. Just did that for my own curiosity. AI sucks! ;).

Anyways, I'll think about a solution over the next few weeks. I have finals this week/next so I don't expect to make much real progress soon.
